corehq/toggles.py

```diff
@@ -2026,3 +2026,12 @@ def _commtrackify(domain_name, toggle_is_enabled):
     TAG_SOLUTIONS_LIMITED,
     namespaces=[NAMESPACE_DOMAIN]
 )
+
+
+USH_CASE_CLAIM_ADDONS = StaticToggle(
```

Comment:
I thought we were going to rename the existing `CASE_CLAIM_AUTOLAUNCH` flag, rather than have two USH-specific feature flags.

Reply:
We discussed this a bit on the ticket https://dimagi-dev.atlassian.net/browse/USH-732

My understanding was that there are other projects that should be able to use regular case-search via `CASE_CLAIM_AUTOLAUNCH` flag and additional case-search config functionality on top of the features provided by `CASE_CLAIM_AUTOLAUNCH` will be under the new  `USH_CASE_CLAIM_ADDONS` flag. You can see the discussion under the ticket. Do let me know if my understanding is incorrect.

Reply:
Regular case claim is under `SYNC_SEARCH_CASE_CLAIM`, only USH projects are using `CASE_CLAIM_AUTOLAUNCH`

Reply:
Cool. I have updated now.